src/unit.rs

```diff
@@ -0,0 +1,257 @@
+/// Macro to implement a set of [measurement units][measurement]. Note that this macro must be
+/// executed in direct submodules of the module where the [`system!`](macro.system.html) macro was
+/// executed. `@...` match arms are considered private.
+///
+/// * `$unit`: Unit name (e.g. `meter`, `foot`).
+/// * `$conversion`: Conversion (coefficient and constant factor) from the unit to the base unit of
```

Comment:
The wording of "constant factor" did puzzle me a bit during the first reading  "factor" suggests a product where a sum is computed. AFAIU, this is something like an offset or an intercept?

Reply:
Most units just use a coefficient. The constant factor is for thermodynamic temperature. Thoughts on better wording?

https://github.com/iliekturtles/uom/blob/1a3386a7f999c6da5a366366b094028848795ed2/src/si/thermodynamic_temperature.rs#L90-L93